mitm/mitm.d.ts

```diff
@@ -0,0 +1,44 @@
+// Type definitions for mitm v1.3.0
+// Project: https://github.com/moll/node-mitm
+// Definitions by: Alejandro Sánchez <https://github.com/alejo90>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+///<reference path="../node/node.d.ts"/>
+
+declare module 'mitm' {
+	import * as http from 'http';
+	import * as net from 'net';
+
+	export interface SocketOptions {
+		port: number;
+		host?: string; 
+		localAddress?: string;
+		localPort?: string; 
+		family?: number; 
+		allowHalfOpen?: boolean;
+	}
+
+	export interface BypassableSocket extends net.Socket {
+		bypass(): void
+	}
+
+	type SocketConnectCallback = (socket: BypassableSocket, opts: SocketOptions) => void;
+
+	type SocketConnectionCallback = (socket: net.Socket, opts: SocketOptions) => void;
+	
+	type HttpCallback = (request: http.IncomingMessage, response: http.ServerResponse) => void;
+
+	type Event = 'connect' | 'connection' | 'request';
+
+	type Callback = SocketConnectCallback | SocketConnectionCallback | HttpCallback;
+
+	export interface Mitm {
+		disable(): void;
+		on(event: Event, callback: Callback): void;
+		on(event: 'connect', callback: SocketConnectCallback): void;
+		on(event: 'connection', callback: SocketConnectionCallback): void;
+		on(event: 'request', callback: HttpCallback): void;
+	}
+
+	export default function (): Mitm;
```

Comment:
https://github.com/moll/node-mitm/blob/master/index.js#L15
this is not default export.
you shoud use `export function _(): Mitm; export = _;` style.